src/workflow/ArcanistPatchWorkflow.php

```diff
@@ -835,6 +845,15 @@ public function run() {
     return 0;
   }
 
+  // Default value for $recursive = true so we do not break existing behavior
+  private function submoduleUpdate($repository_api, $recursive = true) {
+    $command = 'submodule update --init';
+    if($recursive)
+      $command .= ' --recursive';
```

Comment:
Wrap this in braces? I know that it works as-is, but for sake of clarity, it may be worth it.